ios/COVIDSafePaths.xcodeproj/project.pbxproj

```diff
@@ -2234,9 +2234,9 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "org.pathcheck.covid-safepaths";
+				PRODUCT_BUNDLE_IDENTIFIER = org.pathcheck.bt;
 				PRODUCT_NAME = BT;
-				PROVISIONING_PROFILE_SPECIFIER = "match Development org.pathcheck.covid-safepaths";
```

Comment:
Is this provisioning profile switch meant to be included?

Reply:
nope!